specification/cognitiveservices/data-plane/ComputerVision/stable/v3.2/ComputerVision.json

```diff
@@ -0,0 +1,1922 @@
+          "type": "string",
+          "x-nullable": true
+        },
+        "words": {
+          "description": "An array of objects, where each object represents a recognized word.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/OcrWord"
+          },
+          "x-nullable": true
+        }
+      },
+      "x-nullable": true
+    },
+    "OcrWord": {
+      "description": "Information on a recognized word.",
+      "type": "object",
+      "properties": {
+        "boundingBox": {
+          "description": "Bounding box of a recognized word. The four integers represent the x-coordinate of the left edge, the y-coordinate of the top edge, width, and height of the bounding box, in the coordinate system of the input image, after it has been rotated around its center according to the detected text angle (see textAngle property), with the origin at the top-left corner, and the y-axis pointing down.",
+          "type": "string",
+          "x-nullable": true
+        },
+        "text": {
+          "description": "String value of a recognized word.",
+          "type": "string",
+          "x-nullable": true
+        }
+      },
+      "x-nullable": true
+    },
+    "TagResult": {
+      "description": "The results of a image tag operation, including any tags and image metadata.",
+      "type": "object",
+      "properties": {
+        "tags": {
+          "description": "A list of tags with confidence level.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ImageTag"
+          },
+          "x-nullable": true
+        },
+        "requestId": {
+          "description": "Id of the REST API request.",
+          "type": "string",
+          "x-nullable": true
+        },
+        "metadata": {
+          "$ref": "#/definitions/ImageMetadata"
+        },
+        "modelVersion": {
+          "$ref": "#/definitions/ModelVersion"
+        }
+      },
+      "x-nullable": true
+    },
+    "AreaOfInterestResult": {
+      "description": "Result of AreaOfInterest operation.",
+      "type": "object",
+      "properties": {
+        "areaOfInterest": {
+          "$ref": "#/definitions/BoundingRect",
+          "description": "A bounding box for an area of interest inside an image.",
+          "readOnly": true
+        },
+        "requestId": {
+          "description": "Id of the REST API request.",
+          "type": "string",
+          "x-nullable": true
+        },
+        "metadata": {
+          "$ref": "#/definitions/ImageMetadata"
+        },
+        "modelVersion": {
+          "$ref": "#/definitions/ModelVersion"
+        }
+      },
+      "x-nullable": true
+    },
+    "ImageUrl": {
+      "required": [
+        "url"
+      ],
+      "type": "object",
+      "properties": {
+        "url": {
+          "description": "Publicly reachable URL of an image.",
+          "type": "string"
+        }
+      }
+    },
+    "ComputerVisionErrorResponse": {
```

Comment:
Why do we need this wrapper, can't we expose `ComputerVisionError` directly?

Reply:
@jhendrixMSFT this one is the wrapper that contains `error`, whereas the second one contains `code`, `message`